build.gradle.kts

```diff
@@ -3,160 +3,51 @@
 import kotlinx.benchmark.gradle.KotlinJvmBenchmarkTarget
 
 plugins {
-    kotlin("multiplatform")
-    kotlin("plugin.serialization")
+    buildsrc.conventions.lang.`kotlin-multiplatform-js`
```

Comment:
I like the longer names because they're less ambiguous, but `buildsrc.plugins.kmpjs` works too 👍 

The plugin ID for precompiled script plugins is determined by the package and filename, so it can be anything (so long as it doesn't clash with an existing plugin ID).

Reply:
I've renamed them to have a dash between 'kmp' and the target, e.g. `buildsrc.plugins.kmp-js`, because I thought a name like `buildsrc.plugins.kmpbase ` is a little more unclear than `buildsrc.plugins.kmp-base`, and I wanted to use the same naming pattern for all of them. But it's really not that important and if you want it to be `buildsrc.plugins.kmpjs` then I can change it again :)